### PR TITLE
Introducing checkqc

### PR DESCRIPTION
**Pull request by matrulda**
**What problems does this PR solve?**
Replaces the sisyphus quickReport + QC check with checkQC.

**How has the changes been tested?**
New action tested locally with checkQC web service.  

**Reasons for careful code review**
If any of the boxes below are checked, extra careful code review should be inititated.

  - [ ] This PR contains code that could remove data

### CODE REVIEW

File: actions/run_checkqc.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            return response
+        except RequestException as err:
+            self.logger.error("An error was encountered when "
+                              "querying url: {0},  {1}".format(url, err))
+            raise err
+
+    def run(self, url, ignore_result, verify_ssl_cert):
+        response = self.query(url, verify_ssl_cert).json()
+        exit_status = response["exit_status"]
+        if (ignore_result & exit_status != 0):
```
Comment (johandahlberg) [inline actions/run_checkqc.py]: The `&` operator in Python means [bitwise and](https://wiki.python.org/moin/BitwiseOperators), so this should be `and` here.

File: actions/run_checkqc.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            self.logger.warning("Ignoring the failed result because of override flag.")
+            return True, response
+        elif (exit_status == 0):
```
Comment (johandahlberg) [inline actions/run_checkqc.py]: No need for the parenthesis here.

File: actions/run_checkqc.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+name: get_response
```
Comment (johandahlberg) [inline actions/run_checkqc.yaml]: I think that the name of this action should be `run_checkqc`, and then the name of the python script should probably be the same to make it easier to see which things belong together.

File: tests/test_run_checkqc.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
+import mock
+import json
+import requests
+
+from st2tests.base import BaseActionTestCase
+
+from run_checkqc import runCheckQC
+
+
+class runCheckQCTestCase(BaseActionTestCase):
```
Comment (johandahlberg) [inline tests/test_run_checkqc.py]: Capital letter in the class name here: `RunCheckQCTestCase`

File: actions/run_checkqc.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+import requests
+import json
+from requests.exceptions import RequestException
+
+# Needs to be run in a Stackstorm virtualenv
+from st2actions.runners.pythonrunner import Action
+
+class runCheckQC(Action):
```
Comment (johandahlberg) [inline actions/run_checkqc.py]: Capital letter in python class names: `RunCheckQC`

**Review by johandahlberg — CHANGES_REQUESTED** [review]
This looks great! I have added some small comments, mostly on syntax things.

**Review by monikaBrandt — COMMENTED** [review]
Looks good! Good job!

**Review by johandahlberg — APPROVED** [review]

**Comment by matrulda** [comment]
@johandahlberg I've fixed your requested changes. 

@monikaBrandt @johandahlberg Thank you! We might wanna wait with merging until this branch has been validated.

**Comment by johandahlberg** [comment]
Let us know when you want this merged @MatildaAslin .

**Comment by matrulda** [comment]
Validation complete! 🎉 Feel free to merge @monikaBrandt @johandahlberg

**Comment by johandahlberg** [comment]
🍾 Great!

**Pull request merged**